plugins/doc_fragments/postgres.py

```diff
@@ -48,6 +48,11 @@ class ModuleDocFragment(object):
       - If the file exists, the server's certificate will be verified to be signed by one of these authorities.
     type: str
     aliases: [ ssl_rootcert ]
+  connect_params:
+    description:
+      - Any additional parameters to be passed to libpg as kwargs.
```

Comment:
```suggestion
      - Any additional parameters to be passed to libpg as kwargs.
```
Does `as kwargs` say anything to users? If not, let's remove it maybe?

Reply:
Besides that LGTM (i hope there will not be conflicts with existing parameters like when `login_host` and connection_params' `host` will be both passed

Reply:
> Does `as kwargs` say anything to users? If not, let's remove it maybe?
Yes, that may be incomprehensible. I removed it.

Reply:
> i hope there will not be conflicts with existing parameters like when `login_host` and connection_params' `host` will be both passed

Parameters passed via `connect_params` override other parameters. 

In this case `login_host` is mapped to `host` param for libpg and `connect_params['host']` overrides the `host` param (for libpg). So `connect_params['host']` is taken as hostname.

Maybe we should leave a note in the description like this:?
`- These parameters take precedence over the others.`

Reply:
sounds good, wfm, thanks!